File: sunjiayuan_web/src/main/webapp/js/service/carService.js
//服务层
app.service('carService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../car/findAll.do');		
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../car/findPage.do?page='+page+'&rows='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../car/findOne.do?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../car/add.do',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../car/update.do',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../car/delete.do?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../car/search.do?page='+page+"&rows="+rows, searchEntity);
	}    	
});

File: sunjiayuan_dao/src/main/java/com/sunjiayuan/service/CarService.java
package com.sunjiayuan.service;
import java.util.List;
import com.sunjiayuan.pojo.TbCar;

import com.sunjiayuan.pojo.PageResult;
import com.sunjiayuan.pojo.TbUser;

/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface CarService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<TbCar> findAll();
	
	
	/**
	 * 返回分页列表
	 * @return
	 */
	public PageResult findPage(int pageNum, int pageSize);
	
	
	/**
	 * 增加
	*/
	public void add(TbCar car);
	
	
	/**
	 * 修改
	 */
	public void update(TbCar car);
	

	/**
	 * 根据ID获取实体
	 * @param id
	 * @return
	 */
	public TbCar findOne(Long id);
	
	
	/**
	 * 批量删除
	 * @param ids
	 */
	public void delete(Long[] ids);

	/**
	 * 分页
	 * @param pageNum 当前页 码
	 * @param pageSize 每页记录数
	 * @return
	 */
	public PageResult findPage(TbCar car, int pageNum, int pageSize);

    void addCar(List<TbCar> cars, TbUser user);
}
